.circleci/config.yml

```diff
@@ -137,15 +141,21 @@ jobs:
   test-browser:
     <<: *test-defaults
     environment:
-      # browser.test_sdl_ttf_render_text_solid depends on Arial from Microsoft
       # browser.test_sdl2_mouse and/or SDL2 should be fixed. The case happens
```

Comment:
the comments about `test_sdl2_mouse` and `test_html5_webgl_create_context` may have been removed by mistake, I think they are still needed?